Dockerfile

```diff
@@ -1,13 +1,4 @@
-FROM ubuntu:18.04
-RUN set -xe \
-    && apt-get update \
-    && apt-get install -y curl gcc g++ \
-    && apt-get install -y python3-dev \
-    && apt-get install -y libffi-dev \
-    && apt-get install -y libssl-dev \
-    && apt-get install -y python3-pip \
-    && apt-get install -y git
-RUN pip3 install --upgrade pip
+FROM python:3.7-slim-buster
```

Comment:
This will probably break the deployed version since git (and maybe curl) are not longer installed. This project requires git being installed.

Reply:
Also, MAKE-1075 is open to do this.

Reply:
Good catch, I've added git/curl back in. Is curl just there as a transitive dependency for git though, or do we use it somewhere else?

@lydiastepanek I think I missed MAKE-1075 since it was in this sprint assigned to you and there's no link in the tix. Do you want to do a quick PR for that ticket and I'll rebase, or should I just take that one as well and do both in this PR?

Reply:
Feel free to take MAKE-1075 and assign to yourself.

Reply:
git is definitely required. I'm not sure about curl. I know we need it in other tools (to install node, I think), but we might not need it here.